### ISSUE TITLE
Celebrate our 3,000th Issue and Pull Request

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#2982]
As [is](https://github.com/Seneca-CDOT/telescope/issues/500), [our](https://github.com/Seneca-CDOT/telescope/issues/1000) [practice](https://github.com/Seneca-CDOT/telescope/issues/1968), we try to pause every time we hit a major milestone in the project and celebrate our community and what we've done together.  Soon we'll hit 3,000.

As I write this, we have:

- 148 Contributors
- 129 Open Issues
- 2,720 Commits
- 1,587 Closed PRs

It's a tradition for us to add more [Unsplash photos](https://unsplash.com/) to commemorate our achievement. Please paste some links below to Unsplash photos that you think represent what it's like to work on Telescope, what we're doing together

Once we have them all in here, I'll add them to the Image service.

Also, if you have any quotes you want to include in the student quotes, or notice something you saw in another student's blog, please add that too.

**Comment by tcvan0707** [comment]
"If you cannot do great things, do small things in a great way". 😊

**Comment by cindyledev** [comment]
"Do you know what we go through while coding? They [Digital Ocean] know we need good quality [Hacktoberfest t-shirts]. All the blood, sweat, and mainly my tears..." 

-- Amasia Nalbandian

**Comment by manekenpix** [comment]
New image: https://unsplash.com/photos/-Xqckh_XVU4

![image](https://user-images.githubusercontent.com/23108901/155031982-0632cd31-199d-4867-8f8a-d41587e2e2fb.png)

**Comment by humphd** [comment]
I like: https://unsplash.com/photos/zba0XM1kIEQ

Fitting all the different pieces together to make something that will last.

![](https://images.unsplash.com/photo-1616232218232-8461ad93412e?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2371&q=80)

**Comment by JerryHue** [comment]
I wonder if we could do something somewhat special. I've been looking at three.js for a while, so I was wondering if we could have a hero banner that uses three.js animations. To make it special, the hero banner can appear randomly, with a really low chance of appearing, like an [easter egg](https://en.wikipedia.org/wiki/Easter_egg_(media)). 😄 

I think a CSS animation could also be good, to make it more lightweight.

**Comment by Kevan-Y** [comment]
https://unsplash.com/photos/03ztkGdYWU8.
There is a bunch of unknown part in telescopes, but we stand out in one part.
![image](https://user-images.githubusercontent.com/58233223/155530972-fd11eed6-a185-4e64-8f81-86a7d22f74f9.png)

**Comment by Kevan-Y** [comment]
> JerryHue

We can use something like that https://spline.design/ to generate an iframe.

**Comment by tpmai22** [comment]
@sirinoks you can put your opinion about this issue here

**Comment by sirinoks** [comment]
On the starfield idea (Generating a starfiled where each star is a GitHub profile picture of our contributors) - I don't think it's a good one. Here is why.

I roughly understand what needs to be done: 
- Put profile pictures as a star shape
- Shift transparency from complete white, to then showing the profile pictures (CSS opacity)
- Change zoom level of the stars with time

I can see this looking very bad and wrong. Like one of those old websites that try to invent something new, but look terrible in the end. It can be done well, but that would be difficult. I don't know if we have someone here who's experienced in CSS animations. 

In conclusion, it requires a lot of effort to be done properly and look nice. If we don't do it properly, it will look cringe for the lack of a better word. If someone thinks this is cool and they are capable to implement it - sure. If not - forget the idea.
